lib/scrabble/dictionary.rb

```diff
@@ -1,4 +1,17 @@
 module Scrabble
 	class Dictionary
+
```

Comment:
It looks like your text editor is set to use hard tabs (that is, the actual tab character) instead of soft tabs (spaces). That can cause your code to look weird when displayed elsewhere, like here apparently Github renders a tab as 8 spaces, which probably isn't what you want. Your editor should be configurable to use 2 spaces (per Ruby convention) in place of tabs.